test/example/find_div0s.cpp

```diff
@@ -45,7 +46,8 @@ int main() {
         // Check if the right operand is a constant, and if it is 0
         auto right = curr->right->dyn_cast<Const>();
         if (right && right->value.getInteger() == 0) {
-          std::cout << "We found that " << curr->left << " is divided by zero\n";
+          std::cout << "We found that ";
+          printWasm(curr->left, std::cout) << " is divided by zero\n";
```

Comment:
Almost, the order would be wrong. What is really needed is to overload the `<<` operator I suppose, in the `std` namespace? I'm always wary about doing that...